package/MDAnalysis/core/groups.py

```diff
@@ -3051,46 +3052,7 @@ def cmap(self):
                 "cmap only makes sense for a group with exactly 5 atoms")
         return topologyobjects.CMap(self.ix, self.universe)
 
-    def convert_to(self, package):
-        """
-        Convert :class:`AtomGroup` to a structure from another Python package.
-
-        Example
```

Comment:
We'd like to keep this documentation somewhere.

Reply:
it's currently in the `__call__` method of the `ConverterWrapper` and can be seen with `u.atoms.convert_to?`